docs/css/style.css

```diff
@@ -16,6 +16,13 @@ h1, h2, h3, h4 {
   letter-spacing: 2px;
 }
 
+.darkBorder {
```

Comment:
This is a matter of opinion, but I'm not a huge fan of the black border around the list of previous posts.  It might look better with no border or perhaps a slightly different color background.  I can live with it as is if everyone else prefers it that way.